extension/lib/display.js

```diff
@@ -68,25 +68,17 @@ class Display {
     Return: 
     The total time remaining to read document, based on avg letters/word and lowball estimate of WPM (int)
     */
-    initTimer(readableDomIds) {
-        let total_char = 0; // NEED TO REFACTOR TOTAL_CHAR CALCULATION
-        let avg_letters_per_word = 6; // Actual is 4.79 (from Quora), add a space after each word to get 6. 
+    initTimer(readableDomIds, total_words) {
         let avg_read_speed = 200; // Low ball estimate, from Rayner (in WPM)
```

Comment:
been meaning to ask in the previous cl, so ill ask while i remember.
Q: why don't we estimate using the actual speed currently set by the user? wouldn't that would be the most faithful estimation?
-- If user has not changed speed at all, then shouldn this just be the default speed?

Reply:
If non trivial code, feel free to leave TODO for a future CL

Reply:
So we have the auto-scroll start at 400 WPM (-ish; again it's just an estimate), but we calculate the reading time with a low-ball estimate of 200 WPM. 200 WPM is actually almost dreadfully slow (assuming they read at a constant speed, which the autoscroll does). I suspect the 200 WPM is more of a result of a distraction than reading speed, so I think it is valid to incorporate it in a realistic "total time til yer done", but the auto-read should be faster than that, since that is supposed to measure literally how many words go by per second, not accounting for things like distractions, and it is only useful if it is around 400 WPM

Reply:
But good point that if a user is using the autoscroll we will want to adjust the estimated time

Reply:
Q1. Do we agree that "if in autoscroll, when user increase his wpm, then the estimated time should decrease?"
This means that we want estimated time during autoscroll = words left / current autoscroll wpm

Q2. If Q1 is a yes, then _outside_ of autoscroll, what wpm should we use for estimated time?
I just assumed it should be autoscroll wpm. If we use other wpm's, then as soon as user presses alt, the estimated time would suddenly decrease by a lot, which might be confusing to the user.

And... maybe your comment means that the default autoscroll wpm should be 200 wpm. It was unbearable slow before the animation, but w/ the animation, maybe it's more tolerable?

Reply:
Ikk I am not sure about this predicament; perhaps more user testing would elucidate

Though I don't think we should move autoscroll down to 200 for the purpose of the timer, I think the current speed is actually pretty good. (My mom was actually significantly faster than it and immediately sought out how to make it faster)